README.md

```diff
@@ -56,6 +61,33 @@ elao_grafana_config:
     - admin_password: admin
 ```
 
+Configure datasources :
+
+```yaml
+elao_grafana_datasources:
+  - name: influxdb
+    type: influxdb
+    isDefault: true
```

Comment:
I need to follow Grafana API naming -> camelCase
See http://docs.grafana.org/reference/http_api/#get-all-datasources